mkosi/__init__.py

```diff
@@ -6052,6 +6052,50 @@ def virt_name(args: CommandLineArguments) -> str:
     return cast(str, name[:13])
 
 
+def has_networkd_vm_vt() -> bool:
+    networkd_search_paths = run(["systemd-path", "systemd-search-network"], stdout=PIPE, text=True).stdout
+    for path in networkd_search_paths.split(":"):
+        if os.path.exists(os.path.join(path, "80-vm-vt.network")):
+            return True
+
+    return False
+
+
+def ensure_networkd(args: CommandLineArguments) -> None:
+    networkd_is_running = run(["systemctl", "is-active", "--quiet", "systemd-networkd"], check=False).returncode == 0
+    if not networkd_is_running:
+        die("--network-veth requires systemd-networkd to be running (systemctl enable --now systemd-networkd)")
```

Comment:
To bring it in line with other messages
```suggestion
        die("--network-veth requires systemd-networkd to be running (`systemctl enable --now systemd-networkd`)")
```